app/frontend/src/utils/constants.js

```diff
@@ -20,6 +20,17 @@ export const FieldValidations = Object.freeze({
   PASSWORD_MAX_LENGTH: 60
 });
 
+export const KcEnv = Object.freeze({
+  DEV: 'DEV',
+  TEST: 'TEST',
+  PROD: 'PROD'
+});
+
+export const KcClientStatus = Object.freeze({
+  AVAILABLE: 'Available',
+  NOT: 'Not Available'
+});
```

Comment:
Are we intending on usins these constants elsewhere in the codebase? I saw you refactored this out of the component. Mixed feelings on whether the extraction is needed or not.

Reply:
Yeah I'd refactored it out and was using it as a comparison, so it was appropriate. But then I changed implementation.
Let's leave it for now, and once I wrap all the other code for this I'll see if it's used. 
Added a TODO comment